### ISSUE TITLE
[MODIFICATION] Use new caiman data storage dir for mcorr outputs

### ISSUE THREAD

**Issue by kushalkolar** [nel-lab/mesmerize-core#4]
Stop using the `kushalkolar` branch for saving the mcorr memmaps with a specific filename, use this once the PR is merged into `caiman` `master` branch.

https://github.com/flatironinstitute/CaImAn/pull/948

**Comment by kushalkolar** [comment]
done in #22 

see: https://github.com/flatironinstitute/CaImAn/pull/978
